lessonFive.py

```diff
@@ -17,33 +17,50 @@
 # only if the length of the list is less than 5.
 def addToList(element) :
     # your code here
-
+    if len(myList) <= 5:
+        myList.append(element)
+    return myList
 
 # Complete a function that removes the last element from the list
 # 'myList', and returns it unless there is one left, then leave it.
 def removeFromList() :
     # your code here
-
-
+    if len(myList) > 1:
+        myList.pop()
+        return myList
+    else:
+        return myList
 
 # Complete a function that returns the index of 'element'.
 # If the list doesn't contain the element then return -1.
 def getIndex(element) :
     # your code here
-
-
+    for item in myList:
+        if item != element:
```

Comment:
this is going to return -1 a lot. It's kind of like the same thing you fixed in the last exercise